### DISCUSSION TITLE
Who removes remote branch?

### DISCUSSION THREAD

**Discussion by git-hls-demo** [community/community#21961] [Pull Requests]
<p>Hi,</p>
<p>When collaborating on a larger project or working for an organization and a pull-request is made linked to an issue, who usually removes the remote branch upon successful merge? Is it the reviewer/administrator at the time of merge or is the collaborator/employee responsible for doing this? Thanks.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>With a shared repository I’d usually expect the collaborator to clean up after themselves. With a fork they have to anyway, because the fork is their repository. :wink:</p>
<p>But that’s opinion, in the end it’s up to those in charge of the shared repository how they want to organize things.</p>

**Comment by completerevolution** [comment]
<p>Use the option which allows automatic removal of the branch upon merge.</p>

**Comment by git-hls-demo** [comment]
<p>Great, thank you very much.</p>

**Comment by git-hls-demo** [comment]
<p>I didn’t know that was possible. Good idea, thanks.</p>
